### DISCUSSION TITLE
change slug of sidebar item url

### DISCUSSION THREAD

**Discussion by pincman** [facebook/docusaurus#7594] [Q&A]
example

the docs files in `docs/manuals` and `docs/tutorials`,and config

```js
const sidebars = {
    courses: [
        {
            type: 'autogenerated',
            dirName: 'tutorials',
        },
    ],
    docs: [
        {
            type: 'autogenerated',
            dirName: 'manuals',
        },
    ],
}
```

how to config it if I want to view doc files in  dir `/docs/manuals`  by url '/docs/contents` ?

**Comment by slorber** [comment]
You have to use `slug: /content/myDoc` in each doc of the sidebar

Note: 2 docs in the same folder can have completely different slugs, they do not need to have the same "path prefix"

---

Eventually, we could add a category metadata option so that you could just write  `docs/manuals/_category_.json` with `slug: "content"`

But I think it would require a serious refactor, as category metadata files are currently read after we compute all doc slugs.
